### PR TITLE
Improve pool provider status messages

### PR DESCRIPTION
**Pull request by MattGal**
Remove " or does not exist." as well as if the work was submitted to the point of having a correlation id, the work exists

### CODE REVIEW

File: src/Controllers/PoolProviderController.cs [hunk 1]
```diff
@@ -376,17 +376,17 @@ public async Task<IActionResult> GetAgentRequestStatus([FromBody] AgentRequestSt
                 case "unscheduled":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Helix work item in job {correlationId} for agent {workItemId} is currently {workItemDetails.State} or does not exist."
+                        statusMessage = $"(Work has been submitted to the Helix API, but has not started yet) Helix work item in job {correlationId} for agent {workItemId} is currently {workItemDetails.State}"
```
Comment (dougbu) [inline src/Controllers/PoolProviderController.cs:379]: nit: Suggest you flip this around and put the original details in parentheses.

File: src/Controllers/PoolProviderController.cs [hunk 2]
```diff
@@ -376,17 +376,17 @@ public async Task<IActionResult> GetAgentRequestStatus([FromBody] AgentRequestSt
...
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Helix work item in job {correlationId} for agent {workItemId} is currently waiting for a machine."
+                        statusMessage = $"(Work has been submitted to the Helix API, and is waiting for a machine) Helix work item in job {correlationId} for agent {workItemId} is currently waiting for a machine."
                     });
                 case "failed":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Helix work item in job {correlationId} for agent {workItemId} failed.  Please check the logs."
+                        statusMessage = $"(Error state: Please contact dnceng with Job Id) Helix work item in job {correlationId} for agent {workItemId} failed.  Please check the logs."
```
Comment (dougbu) [inline src/Controllers/PoolProviderController.cs:389]: nit: Oh nos❕ Extra space between sentences 😈

Comment (MattGal) [reply src/Controllers/PoolProviderController.cs]: That's only been the grammar rule for a couple years but no problem :)

**Comment by Pilchie** [comment]
Also tagging @davidfowl and @HaoK

**Comment by MattGal** [comment]
OK this has been open a while, the builds succeeded, going to merge and close.  Feel free to PR more useful messages here, or even bring some of your ASP expertise to examine the project.

**Pull request merged**